ViewControllers/Home/HomeViewController.swift

```diff
@@ -65,7 +62,7 @@ final class HomeViewController: BaseViewController {
             guard let self = self else{
                 return
             }
-//            TO-DO
+            self.showAlbumDetails(withAlbum: album)
```

Comment:
if you want you can also remove guard and replace it with
`self.map { $0.showAlbumDetails(withAlbum: album) }`